<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-U8K0EIF" failures="1" tests="22" name="runner.TestRunner" time="621.554" errors="1" timestamp="2025-09-13T19:11:20 IST" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="11.459"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="23.382"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="21.466"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="30.847"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="34.825"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="36.029"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="45.088"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="34.535"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="6.718"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="103.877">
    <failure message="Login failed! expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Login failed! expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepdefinitions.Subscription_Payment_Cancel_Stepdef.user_is_logged_in_for_payment_cancel(Subscription_Payment_Cancel_Stepdef.java:27)
at ✽.the user is logged in for payment cancel(file:///C:/Users/HP/eclipse-workspace/nobrokertestapplication/src/test/resources/features/subscriptionpaymentcancel.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="6.209"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="69.238"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="8.016"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="33.679"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="18.920"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="22.706"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="8.207">
    <error message="stale element reference: stale element not found in the current frame
  (Session info: chrome=140.0.7339.81)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: &#039;4.34.0&#039;, revision: &#039;707dcb4246*&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;21.0.7&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [e4a6f9729a6f02465ed498df1d940fd0, clickElement {id=f.53C3338F299954E7EC246311F6FC9FAA.d.982170F45CDF4B6CCDB1C9B5ED2EE526.e.103}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.81, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\HP\AppData\Local\T...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50185}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50185/devtoo..., se:cdpVersion: 140.0.7339.81, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (e4a6f9729a6f02465ed498df1d940fd0)] -&gt; xpath: //div[text()=&#039;Chennai&#039;]]
Session ID: e4a6f9729a6f02465ed498df1d940fd0" type="org.openqa.selenium.StaleElementReferenceException">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: stale element not found in the current frame
  (Session info: chrome=140.0.7339.81)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: '4.34.0', revision: '707dcb4246*'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [e4a6f9729a6f02465ed498df1d940fd0, clickElement {id=f.53C3338F299954E7EC246311F6FC9FAA.d.982170F45CDF4B6CCDB1C9B5ED2EE526.e.103}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.81, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\HP\AppData\Local\T...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50185}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50185/devtoo..., se:cdpVersion: 140.0.7339.81, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (e4a6f9729a6f02465ed498df1d940fd0)] -> xpath: //div[text()='Chennai']]
Session ID: e4a6f9729a6f02465ed498df1d940fd0
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:544)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at pages.PropertySearch.selectCity(PropertySearch.java:49)
at stepdefinitions.HistoryStepDef.the_user_has_previously_searched_for(HistoryStepDef.java:29)
at ✽.the user has previously searched for "Adyar, Chennai"(file:///C:/Users/HP/eclipse-workspace/nobrokertestapplication/src/test/resources/features/history.feature:4)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.TestRunner" name="runScenario" time="9.172"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="46.900"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="10.425"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="19.125"/>
  <testcase classname="runner.TestRunner" name="runScenario" time="20.731"/>
</testsuite> <!-- runner.TestRunner -->
